tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -48,12 +48,14 @@ type cluster struct {
 	stressKeySize        int
 	stressKeySuffixRange int
 
-	Size      int
-	Stressers []Stresser
+	Size           int
+	Stressers      []Stresser
+	LeaseStressers []Stresser
```

Comment:
The reason I have `LeaseStressers ` is because the lease checker needs to know about lease stressers https://github.com/fanminshi/etcd/blob/42e31c8c3850368293d14411015d4c23305b6079/tools/functional-tester/etcd-tester/main.go#L126-L137